ui/app/styles/components/form-section.scss

```diff
@@ -7,7 +7,8 @@
   }
 }
 
-.field:first-child .form-section {
+.field:first-child .form-section,
+.box > .field > .field.form-section {
   padding: 0;
   box-shadow: none;
 }
```

Comment:
Seems like this may be widely used. Will this impact other existing forms?

Reply:
From when I poked around it seemed like we often try to remove any form-section styling when things are nested within multiple field divs (because it's inside a toggle)

I poked around and didn't see anywhere else that looked affected - but I could add `string-list` to increase the specificity?

Reply:
Sure that sounds good